components/director/internal/tenantfetchersvc/resync/event_page.go

```diff
@@ -223,11 +227,19 @@ func constructSubaccountTenant(ctx context.Context, jsonPayload, name, subdomain
 		log.C(ctx).Debugf("Missing or invalid format of region field: %s for tenant with ID: %s", ep.FieldMapping.RegionField, externalTenant)
 	}
 	region := regionField.String()
+
 	parentIDField := gjson.Get(jsonPayload, ep.FieldMapping.GlobalAccountKey)
 	if !parentIDField.Exists() {
 		return nil, invalidFieldFormatError(ep.FieldMapping.GlobalAccountKey)
 	}
 	parentID := parentIDField.String()
+
+	var customerIDValue *string
+	customerIDField := gjson.Get(jsonPayload, ep.FieldMapping.LabelsField).Get(ep.FieldMapping.CustomerIDField)
+	if customerIDFieldArr := customerIDField.Array(); customerIDField.IsArray() && len(customerIDFieldArr) > 0 {
+		customerIDValue = str.Ptr(customerIDFieldArr[0].String())
+	}
```

Comment:
Do we always expect to have an array only with one value?

Reply:
Yes. PR for reference: https://github.wdf.sap.corp/cp-cfs-services/accounts-service/pull/1446/files

Reply:
And a swagger: https://accounts-service.cfapps.eu10.hana.ondemand.com/api?urls.primaryName=Accounts-Service-Api$cis-system&scope=cis-system#/g_Accounts%20APIs/getSubaccount_1